src/main/java/com/jcraft/jsch/JSch.java

```diff
@@ -27,15 +27,15 @@
 package com.jcraft.jsch;
 
 import java.io.InputStream;
-import java.util.Enumeration;
-import java.util.Hashtable;
+import java.util.Map;
 import java.util.Vector;
+import java.util.concurrent.ConcurrentHashMap;
 
 public class JSch {
   /** The version number. */
   public static final String VERSION = Version.getVersion();
 
-  static Hashtable<String, String> config = new Hashtable<>();
+  static Map<String, String> config = new ConcurrentHashMap<>();
```

Comment:
I don't think there is any value in using a `ConcurrentHashMap` instead of a regular `HashMap` since all accesses to the `config` are synchronized on `config` anyway.